bin/docker/mqtt_bridge_run.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set +x
```

Comment:
```suggestion
set +x

# Clean-up /tmp directory from files which might have remained from previous container restart
# We ignore any errors which might be caused by files injected by different agents which we do not have the rights to delete
rm -rfv /tmp/* || true
```

Would you mind to put this as well? We had issues in the past with `/tmp` folder growing out of our control.

Reply:
How come? We are not using this /tmp anywhere 

In the HTTP bridge, it was where the generated config were placed. 
Not the case with the MQTT Bridge, I guess so

Reply:
It was not the HTTP bridge there, it was Vert.x filling the tmp folder. We can't know everything about what our deps are doing.

Reply:
Oh, I see now.